core/src/main/scala/org/ensime/core/ScalapSymbolToFqn.scala

```diff
@@ -51,24 +62,32 @@ trait ScalapSymbolToFqn {
 
     val scalaName = aPackage + "." + name
     val parentPrefix = if (sym.isModule) scalaName + "." else scalaName + "#"
-    val fields = sym.children.collect {
+    val fields: Map[String, RawScalapField] = sym.children.collect {
       case ms: MethodSymbol if !ms.isMethod && ms.isLocal =>
-        rawScalaField(ms, parentPrefix)
-    }
+        val field = rawScalaField(ms, parentPrefix)
+        field.javaName.fqnString -> field
+    }(breakOut)
 
-    val methods = sym.children.collect {
-      case ms: MethodSymbol if ms.isMethod =>
+    val methods: mutable.ArrayBuffer[RawScalapMethod] = sym.children.collect {
```

Comment:
yeah, I guess  it's a remnant of a previous iteration of this PR, should be fixed